src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -66,14 +66,36 @@ def ffmpeg_read(bpayload: bytes, sampling_rate: int) -> np.array:
     return audio
 
 
-def apply_stride(tokens, stride):
-    max_token_n = tokens.shape[-1]
+def audio_to_logits(tokens_or_logits, stride):
+    # Shape is [B, SEQ] for tokens
+    # [B, SEQ, V] for logits
+
+    is_batch_size_one = isinstance(stride, tuple) and not isinstance(stride[0], tuple)
```

Comment:
added this hack to make it work with batch size 1. Maybe we can make this function a bit "prettier"? Or just give it a better docstring?